ods_ci/tests/Resources/Page/Operators/ISVs.resource

```diff
@@ -37,7 +37,8 @@ Install ISV Operator From OperatorHub Via CLI    # robocop: disable
     Wait Until Keyword Succeeds    1 min    0 sec
     ...    Is Resource Present    Subscription    ${subscription_name}     ${namespace}    ${IS_PRESENT}
     Sleep    30s
-    ${installplan_name}=     Get Resource Attribute      ${namespace}
+    ${installplan_name}=       Wait Until Keyword Succeeds    1 min    0 sec
```

Comment:
Isn't 0 sec between the attempts too eager? Shouldn't we give some time between the attempts?

I can see that this is a second instance of such a thing in this file - 3 lines above it was added just recently too [in this change](https://github.com/red-hat-data-services/ods-ci/pull/1963/files#diff-f7e0124fd66571233d7c43584d02497c8e74d0b6e66d77d4876cc1cf0005a23fR37).

So, maybe there is some reason for it and we don't bother, but I truly don't understand why. Personally, I would use something like this instead:

```suggestion
    ${installplan_name}=       Wait Until Keyword Succeeds    1 min    5 sec
```
And same for the waiting for the Subscription thing on the line 37 :shrug: 

Anyway, this is just a proposal - also, I just saw that we have this 0s retry interval on multiple places in other files there too... sooo... whatever.... we're ods-ci is deprecated anyway, right? :rofl:

Reply:
@jstourac this approach did not work so I resolve thread